hugegraph-server/hugegraph-dist/src/assembly/travis/test-check-port.sh

```
@@ -0,0 +1,481 @@
+
+    source "$UTIL_SH"
+    # Force the fallback by pretending ss/netstat don't exist
+    command_available() {
+        if [[ "$1" == "ss" || "$1" == "netstat" ]]; then return 1; fi
+        command -v "$1" >/dev/null 2>&1
+    }
+    # Start a real listener on an ephemeral high port (e.g., 54321)
+    # Use python as a simple cross-platform web server since nc arguments vary
+    python3 -m http.server 54321 --bind 127.0.0.1 >/dev/null 2>&1 &
+    PY_PID=$!
+    trap "kill -9 $PY_PID 2>/dev/null || true" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)